testing/helpers/serverSideDOMAdapterPatch.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+var domAdapter = require("core/dom_adapter");
+var serverSideWindowMock = require("./serverSideWindowMock.js");
+
+exports.set = function() {
+    domAdapter.__restoreOriginal();
+
+    domAdapter.getWindow = function() {
+        return serverSideWindowMock;
```

Comment:
Maybe we must return `serverSideWindowMock.window` instead of `serverSideWindowMock`?